src/main/groovy/de/dkfz/roddy/execution/jobs/GenericJobInfo.groovy

```diff
@@ -6,20 +6,47 @@
 
 package de.dkfz.roddy.execution.jobs
 
-
+import de.dkfz.roddy.config.ResourceSet
 import de.dkfz.roddy.tools.BufferUnit
 import de.dkfz.roddy.tools.TimeUnit
 import groovy.transform.CompileStatic
 
+import java.time.Duration
+import java.time.LocalDateTime
+
 /**
  * Created by michael on 06.02.15.
  */
 @CompileStatic
 class GenericJobInfo {
 
+    ResourceSet askedResources
+    ResourceSet usedResources
     String jobName
     File tool
     String id
+
+    LocalDateTime subTime;
+    LocalDateTime startTime;
+    LocalDateTime endTime;
+
+    String exHosts; // execution hosts
+    String subHost; //submission host
```

Comment:
maybe you could use the full term as variable name instead of abbreviations and comments
(also for other variables)